api/pandas/covariate_dataframe.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: default
+title: Covariates for Places as pd.DataFrame
+nav_order: 3
+parent: Pandas
+grand_parent: API
+---
+
+# Get Covariates for Places as pandas DataFrame
+
+## `datacommons_pandas.build_covariate_dataframe(dcids, stats_vars)`
+
+Returns a `pandas.DataFrame` with [`places`](https://datacommons.org/browser/Place)
+as index and [`stat_vars`](https://datacommons.org/browser/StatisticalVariable)
+as columns, where each cell is latest observed statistic for
+its `Place` and `StatisticalVariable`.
+
+See the [full list of `StatisticalVariable`s](/statistical_variables.html).
+
+**Arguments**
+
+*   `dcids (Iterable of str)`: A list of dcids of the
+    [`Place`](https://datacommons.org/browser/Place)s to query for.
+
+*   `stat_vars (Iterable of str)`: A list of dcids of the
+    [`StatisticalVariable`](https://datacommons.org/browser/StatisticalVariable)s
+    to query for.
+
+**Returns**
+
+A `pandas.DataFrame` with [`places`](https://datacommons.org/browser/Place)
+(str)
+as index and [`stat_vars`](https://datacommons.org/browser/StatisticalVariable)
+(str) as columns, where each cell is latest observed statistic (float) for
+its `Place` and `StatisticalVariable`.
+
+**Raises**
+
+* `ValueError` - If no statistial values found for the given parameters.
+
+Be sure to initialize the library. See the
+[datacommons_pandas library setup guide](/api/pandas/) for more details.
+
+You can find a list of `StatisticalVariable`s with human-readable names [here](/statistical_variables.html).
+
+## Examples
```

Comment:
please add an example for an overly constrained query that results in a value error

Reply:
There are no optional constraints for this function

Reply:
I will add one with no data

Reply:
I added an example for no data for both time series and multivariate data frame.